submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/SubmissionSubmitter.kt

```diff
@@ -56,6 +58,7 @@ open class SubmissionSubmitter(
             request.method
         )
 
+        logger.info { "Saving submission $submission" }
```

Comment:
I'm not sure about this one but, does throwing the object directly in the string display the actual object content?? Sometimes it shows things like `SubmissionRequest@1245` or things like that

Reply:
no, because it is a data class kotlin will automatically add good toString with properties.